trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc

```diff
@@ -0,0 +1,50 @@
+== Die Leistungsfähigkeit der Organisation erhalten
+
+In der Einführung wurde darauf hingewiesen, dass _Trusted Committers_ sowohl technikorientierte als auch organisationsorientierte Verantwortlichkeiten haben.
+Es ist nicht ausreichend, sich nur auf den _Code_ und den Zustand des _Codes_ zu konzentrieren. Um auf lange Sicht erfolgreich zu sein, sollten _Trusted Committer_ auch danach streben, die 
+Leistungsfähigkeit und Motivation der Organisation, welche die Software erstellt, aufrechtzuerhalten. 
```

Comment:
I would probably even change to "... und Motivation der Mitglieder der Organisation die die Software erstellt, ..." as otherwise the agency is attributed to the organization which might differ from the motivation of its individual members.

Reply:
done